scripts/init-sysroot.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+# SPDX-License-Identifier: MIT
+set -xe
+triple="$1"
+srcroot="$2"
+
+sysroot="./system-root/"
+
+install_s() {
+	install --strip-program "tools/cross-binutils/bin/${triple}-strip" "$@"
+}
+
+die() {
+	>&2 tput setaf 1
+	>&2 tput bold
+	for i; do
+		>&2 printf "%s: %s\n" "$0" "$i"
+	done
+	>&2 tput sgr0
+	exit 1
+}
+
+
+amd64_init() {
+	install -dm 750 "${sysroot}/home/managarm"
+	install -d "${sysroot}/boot/managarm"
+	install -d "${sysroot}/boot/grub"
+
+	# install kernel, prekernel
+	_umb="${sysroot}/usr/managarm/bin"
+	install_s -s "${_umb}"/eir-stivale "${sysroot}/boot/managarm"
+	install_s -s "${_umb}"/eir-mb1 "${sysroot}/boot/managarm"
```

Comment:
We should really add `eir-mb2` here too. We build it but it never gets installed. While we're at it, we should update the grub entries to add multiboot 2 options too.